ConfigurationSystem/Client/Utilities.py

```diff
@@ -589,4 +589,24 @@ def getElasticDBParameters( fullname ):
     dbPort = int( result['Value'] )
   parameters[ 'Port' ] = dbPort
 
-  return S_OK( parameters )
\ No newline at end of file
+  result = gConfig.getOption( cs_path + '/UserName' )
```

Comment:
For mysql case the user name is simply in "User", e.g. see line 529 where is
```
result = gConfig.getOption( cs_path + '/User' )
```
so I think you should do the same here.